.github/workflows/full_build.yml

```diff
@@ -0,0 +1,40 @@
+name: Full Build
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
+        with:
+          dotnet-version: 5.0.x
+
+      - name: Restore dependencies
+        run: dotnet restore
+
+      - name: Build
+        run: dotnet build --no-restore
+
+      - name: Test
+        run: dotnet test --no-build --verbosity normal
+
+      - name: Package
+        shell: bash
+        run: |
+          version=$(git describe --tags --abbrev=0)
```

Comment:
What happens if there are two tags on the same commit?

If I am not mistaken this should work as well (taken from https://stackoverflow.com/questions/58177786/get-the-current-pushed-tag-in-github-actions) (already includes the version prefix):

`version=${GITHUB_REF#refs/*/v}`